resources/conf/config.toml.template

```diff
@@ -63,6 +63,26 @@ artifactsDirectory = "/home/wso2/artifacts"
   # Optional path to the private key for Consul communication. If this is set, then you need to also set certFile
   keyFile = "/home/wso2/security/truststore/consul/local-dc-client-consul-0-key.pem"
 
+# Configuration related to the repository where the API artifacts are stored
+[adapter.sourceControl]
+  # Enable/Disable Source Control for API Artifacts
+  enabled = true
+  # Time interval (in seconds) in which the Adapter should fetch updates from the remote repository
+  pollInterval = 30
+  # Time interval (in seconds) in which the adapter should retry fetching artifacts from the remote repository at startup
+  retryInterval = 5
+  # The directory path where the git artifacts are stored
+  artifactsDirectory = "/home/wso2/git-artifacts"
+  # Configurations of the repository
+  [adapter.sourceControl.repository]
+    URL = "https://github.com/username/repository-name.git" # Repository URL where artifacts are stored
+    branch = "main" # Branch of the repository where the artifacts are stored (Use "" for default branch)
+    # Use either username and accessToken, or ssh key for authentication
+    username = "username" # Git username (Use "" in the case of a public repository)
+    accessToken = $env{git_access_token} # Git personal access token or password
+    sshKeyFile = "/home/wso2/.ssh/id_ed25519" # Path to the private key used for authentication (Use "" in the case of a public repository)
```

Comment:
shall we follow same approach used in other configs ?
```

# explanation1
config1
# explanation2
config2
```

Also there's an extra line (`line 85`)